packages/client/src/__tests__/buildNFTAnnotations.test.ts

```diff
@@ -84,16 +86,15 @@ describe('binary', () => {
 describe('dataproxy', () => {
   it('generates no annotations', () => {
     const annotations = buildNFTAnnotations(
-      ClientEngineType.DataProxy,
+      true,
```

Comment:
It is difficult to tell what this boolean does without looking at the func signature.

Reply:
https://www.notion.so/prismaio/Technical-Debt-TS-ORM-c199f16035304e2db13793f176098c24?p=78d58e9a82064605890491722be2f0ff